features/devise_for_user_access.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Logged in user can access article show view
+  As a user
+  In order to gain access to different features in the site
+  I should be able to create an account and login
```

Comment:
Please use the user story I wrote yesterday. This one is simply not entirely relevant